src/main/java/db/DAO/IBaseAirlineCompanyDAO.java

```diff
@@ -0,0 +1,7 @@
+package db.DAO;
```

Comment:
package names should be all lowercase

```suggestion
package db.dao;
```